### DISCUSSION TITLE
Feature Request: Support for Jetstream K/V Store for Session Storage

### DISCUSSION THREAD

**Discussion by izytechAB** [authelia/authelia#8349] [Ideas]
It would be a great improvement if Autelia SSO could support using the Jetstream key-value store for session storage. This seems like a relatively straightforward implementation and would offer better flexibility and scalability for session management.

**Comment by james-d-elliott** [comment]
We plan on adding SQL storage, and we'll keep the other storage options as mainstream and limited as possible.
